app/views/purchases/_form.html.erb

```diff
@@ -1,55 +1,68 @@
 <%= form_with model: @purchase, local: true do |form| %>
 
-  <% if @purchase.errors.any? %>
-    <div id="error_explanation">
-      <h2>
-        <%= pluralize(@purchase.errors.count, "error") %>
-        prohibited this purchase from being saved:
-      </h2>
-      <ul>
-        <% @purchase.errors.full_messages.each do |msg| %>
-          <li><%= msg %></li>
-        <% end %>
-      </ul>
-    </div>
-  <% end %>
-
-
-  <p>
-    <%= form.label :date %><br>
-    <%= form.date_field :date %>
-  </p>
-
-  <p>
-    <%= form.label :description %><br>
-    <%= form.text_field :description %>
-  </p>
-
-  <p>
-    <%= form.label :amount %><br>
-    <%= form.number_field :amount, class: :text_field, step: :any %>
-  </p>
-
-  <p>
-    <%= select_tag(:subcategory_id, grouped_options_for_select(@categories.map do |c|
-        [c.name, c.subcategories.map { |s| [s.name, s.id] }]
-        end), prompt: "Select one!") %>
-  </p>
-
-  <p>
-    <%= form.label :payee %><br>
-    <%= form.text_field :payee %>
-  </p>
-
-  <p>
-      <%= radio_button_tag(:user_id, User.find_by(name: 'Ellen').id) %>
-      <%= label_tag(:user_ellen, "Ellen") %>
-      <%= radio_button_tag(:user_id, (User.find_by(name: 'Ben')).id) %>
-      <%= label_tag(:user_ben, "Ben") %>
-  </p>
-
-  <p>
-    <%= form.submit class: 'button' %>
-  </p>
-
+<% if @purchase.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= pluralize(@purchase.errors.count, "error") %>
+      prohibited this purchase from being saved:
+    </h2>
+    <ul>
+      <% @purchase.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+    </ul>
+  </div>
```

Comment:
The errors could probably do with a bit of styling. 

See [this page](https://bulma.io/documentation/elements/content/), list will look nice if you add the `"content"` class to the wrapping div. (At that point, `h2` may be overkill for the "X prohibited this purchase from being saved" message, could probably simply be a `<p>`).

Might be relevant to use the `is-danger` modifier class: https://bulma.io/documentation/modifiers/syntax/

I've also noticed some error-related bugs, which I'll explain, but I think they're out of scope for the styling PR :)

**Before**
![before](https://user-images.githubusercontent.com/16832997/49426377-4622a280-f805-11e8-9b17-24dbd8bad07e.png)

**Potential after**
![after](https://user-images.githubusercontent.com/16832997/49426376-4622a280-f805-11e8-9b13-3bbbba1b1543.png)

Reply:
Done / ta. Found some error-related bugs while testing things, so have tidied them up in this pass, but a little out of scope.